tf-modules/archive/async_operation.tf

```diff
@@ -19,7 +19,7 @@ resource "aws_ecs_task_definition" "async_operation" {
         "value": "${data.aws_region.current.name}"
       }
     ],
-    "image": "cumuluss/async-operation:27",
+    "image": "cumuluss/async-operation:33",
     "memoryReservation": 700,
```

Comment:
I'm having trouble figuring out if we allow our containers to hit 4GB memory, as this `memoryReservation` entry is the closest thing I can find to a memory limit (although this one is a soft limit). Any idea?

Reply:
Yup, that's what I found. I assume the memory limit is determined by the EC2 cluster instances that we run the tasks on. That's t3.medium for testing, which has 4GB memory.  So there is a possibility of running out of memory, but I think the update to the image just allows node to expand to 4, not just use 4.  We are going to fix the problem at the root for the reconciliation reports.  So this is kind of a bandaid. Mark was not super happy with this, and maybe we should touch base in an arch meeting to make sure we have the tickets we need and if the Dockerfile change should be merged to master or not.